packages/cannon-worker-api/src/cannon-worker-api.ts

```diff
@@ -65,7 +73,7 @@ export class CannonWorkerAPI extends EventEmitter {
     quaternions: Float32Array
   }
 
-  private config: Required<CannonWorkerProps>
+  private config: Required<Omit<CannonWorkerProps, 'frictionGravity'>> & { frictionGravity?: WorldProps['frictionGravity'] }
```

Comment:
This doesn't need to change.

```suggestion
  private config: Required<CannonWorkerProps>
```

Reply:
Hm, I think it does! Otherwise, TS would yell about this `config` assignment: https://github.com/pmndrs/use-cannon/pull/402/files#diff-e3f4dac018aec1d1b23bbbb9ea7d1e0edbdc2dcfee7f35d088d8e145754add61R101, since `config` would be typed with `frictionGravity` as required.

Let me know what you think I'm missing!